jobs/ingestor_syslog/templates/config/input_and_output.conf.erb

```diff
@@ -76,6 +76,42 @@ output {
 
     <% p('logstash_parser.outputs').each do | output | %>
     if [@index_type] == "app" {
+        if [@type] == "app_metric"{
```

Comment:
I don't understand this condition. What is it doing? Why do we need it?

Does it prevent documents with `@index_type` of `app` from being indexed to OpenSearch?